src/main/java/com/ofs/Model/HostGroupJerseyModel.java

```diff
@@ -2,36 +2,68 @@
 
 import java.io.Serializable;
 import java.util.List;
+import java.util.Set;
 
-public class HostGroupJerseyModel   {
+public class HostGroupJerseyModel implements Serializable   {
 	
 	
 	 /**
 	 * 
 	 */
-//	 private static final long serialVersionUID = 1L;
+	 private static final long serialVersionUID = 1L;
 	 int id;
 	 int parentid;
 	 String name;
-	 boolean hostbaseline;
-	 boolean suppress_excluded_service;
-	 boolean inverse_suppression;
-	 boolean host_trap;
-	 boolean send_to_cta;
+	 boolean hostBaseline;
+	 boolean suppressExcludedService;
+	 boolean inverseSuppression;
+	 boolean hostTrap;
+	 boolean sendToCta;
 	 String description;
 	 List<HostGroupJerseyModel> children;
+	 List<IpAddressModel> ipAddress;
+	 Set<String> hostGroupUser;
+	 public Set<String> getHostGroupUser() {
+		return hostGroupUser;
+	}
+
+	public void setHostGroupUser(Set<String> hostGroupUser) {
+		this.hostGroupUser = hostGroupUser;
+	}
+
+	public List<IpAddressModel> getIpAddress() {
+		return ipAddress;
+	}
+
+	public void setIpAddress(List<IpAddressModel> ipAddress) {
+		this.ipAddress = ipAddress;
+	}
+	
 	 
-	 public HostGroupJerseyModel() {}
+	
+
+	public HostGroupJerseyModel() {}
 	 
-	 public HostGroupJerseyModel(int parentid, String name, boolean hostbaseline, boolean suppress_excluded_service, boolean inverse_suppression,boolean host_trap,boolean send_to_cta, String description) {
+	 public HostGroupJerseyModel(int parentid, String name, boolean hostBaseline, boolean suppressExcludedService, boolean inverseSuppression,boolean hostTrap,boolean sendToCta, String description) {
 		
```

Comment:
why constructor overloading ??
If needed both call another constructor to initialize using this(x,y,z,..)